### DISCUSSION TITLE
Is there any difference in the feed of the same channel with different call signs?

### DISCUSSION THREAD

**Discussion by freearhey** [iptv-org/community#525] [Q&A] [answered]
For example, the NewsNet channel has 28 different call signs (https://iptv-org.github.io/?q=%22NewsNet%22%20country:US), and all available streams show the same feed with a slight delay:

KCKS-LD2: https://cdn.igocast.com/channel2_hls/channel2_master.m3u8
W14DK-D2: https://2-fss-2.streamhoster.com/pl_118/204972-1954106-1/playlist.m3u8

I did not find any differences in the program, either:

KCKS-LD2: https://www.tvpassport.com/tv-listings/stations/newsnet-kcksdt2-kansas-city/17543
WYBN-LD8: https://www.tvpassport.com/tv-listings/stations/newsnet-wybnld8-albany-ny/17340
WIVX-LD4: https://www.tvpassport.com/tv-listings/stations/newsnet-wivxld4-cleveland-oh/36510
WQXT-CD6: https://www.tvpassport.com/tv-listings/stations/newsnet-wqxtcd6-jacksonville-fl/17609

The Wikipedia also says that this channel has only two feed versions: domestic and online: https://en.wikipedia.org/wiki/NewsNet#Broadcast_feeds

**Comment by GlamoramaAttack** [comment, marked as answer]
Short answer: there is no difference in the main feed that transports the program. 

Long version: Main difference between "domestic" and "online" feeds is that the "online" feeds don't show the advertisements (outside the US). But that's pointed out in the Wikipedia link too - so you mean probably something different with your question? 

Aside from that all stations must be able to broadcast warning signs for extreme weather alerts in their specific location. 

That doesn't affect the program though but why should there be a difference? The many ABC, CBS etc. stations don't have differences in their programs at prime-time (8-11 pm) either. The original call signs are all from the antenna era. Example KCKS-LD Kansas City is (as your other examples) a low-powered station (only 480p video) for a local market within 100 miles (see: https://www.channelmaster.com/pages/tv-antenna-map-kansas-city-mo-64149)

https://en.wikipedia.org/wiki/KCKS-LD = But with the digital era the call sign could be used for more channels. KCKSDT2 is one of them.

PS: CBS News channels are also localized with no difference in their programs: https://iptv-org.github.io/?q=%22CBSN%22%20country:US  (MSNBC, FoxNews or CNN don't use local call signs).

**Comment by dtngfan85** [comment]
NewsNet has designated opt-outs where local affiliates can insert their own short local news updates and weather reports. Since these are only a few minutes they don't list them as seperate programs in the schedule. Some of the ones that operate on primary .1 channels also pre-empt several hours a week for their FCC mandated community affairs and E/I kids programming, but NewsNet's owner Bridge News is horrible about notifying the various schedule providers about this.

As for CBS News, the local streams are very different compared to the national feed. They simulcast the local CBS Newscasts during mornings, afternoons, evenings and late night, on weekdays they have extended morning and afternoon newscasts that are streaming only, and they carry local breaking news coverage.
